.github/workflows/build.yml

```diff
@@ -113,6 +115,23 @@ jobs:
       with:
         path: ./lite/dist
 
+    # Overwrite some settings to disable cursor blinking,
+    # which causes inadvertent playwright snapshot failures
+    - name: Build the JupyterLite app (in test mode)
+      run: |
+        cd lite
+        rm -r dist # jupyter lite will not build in `dist-test/` if `dist/` exists...?
+        cat jupyter-lite.json | jq '.["jupyter-config-data"].["settingsOverrides"] += {"@jupyterlab/codemirror-extension:plugin":{ "defaultConfig": { "cursorBlinkRate": 0 } }, "@jupyterlab/terminal-extension:plugin": { "cursorBlink": false } }' > jupyter-lite.json.tmp
```

Comment:
Strictly we don't need the `jupyterlab/terminal-extension:plugin` part, but it does not hurt to have it there

Reply:
Good point - I'll get rid of it for clarity before merging :+1: